test/blackbox-tests/test-cases/rule/digest.t/run.t

```diff
@@ -66,5 +66,3 @@ make them more resilient to non semantic changes.
 # need to update it when rule digest version changes.
 
   $ dune build @default
-  running...
```

Comment:
Please remove the CR-someday above.

Reply:
Actually, it's not just the CR-somday that needs updating after your change. Please update the text above too.